docs/source/io_formats/track.rst

```diff
@@ -4,18 +4,27 @@
 Track File Format
 =================
 
-The current revision of the particle track file format is 2.0.
+The current revision of the particle track file format is 3.0.
 
 **/**
 
 :Attributes: - **filetype** (*char[]*) -- String indicating the type of file.
              - **version** (*int[2]*) -- Major and minor version of the track
                file format.
-             - **n_particles** (*int*) -- Number of particles for which tracks
-               are recorded.
-             - **n_coords** (*int[]*) -- Number of coordinates for each
-               particle.
 
 :Datasets:
-           - **coordinates_<i>** (*double[][3]*) -- (x,y,z) coordinates for the
-             *i*-th particle.
+           - **track_<b>_<g>_<p>** (Compound type) -- Particle track information
+             for source particle in batch *b*, generation *g*, and particle
+             number *p*. particle. The compound type has fields ``r``, ``u``,
+             ``E``, ``time``, ``wgt``, ``cell_id``, and ``material_id``, which
+             represent the position (each coordinate in [cm]), direction, energy
+             in [eV], time in [s], weight, cell ID, and material ID,
+             respectively.
+
+             :Attributes: - **n_particles** (*int*) -- Number of
+                            primary/secondary particles for the source history.
+                          - **offsets** (*int[]*) Offset into the array for each
```

Comment:
I'm just working through the code and I'm not 100% sure what the ```offsets``` is. Wondering if other future users might also not understand this from the description.

Reply:
I've updated the description in this file to make it a bit clearer what is meant by offsets. Thanks for the suggestion!